src/renderer/views/Popular/Popular.vue

```diff
@@ -13,6 +13,12 @@
         :data="shownResults"
       />
     </ft-card>
+    <ft-icon-button
+      icon="sync"
+      class="floatingTopButton"
+      :size="10"
```

Comment:
Let's increase the size to 12 so that it looks a little better in the mobile view and give it a theme of `primary` so that it stands out a little better.